vars/RedHat.yml

```diff
@@ -7,12 +7,17 @@ nginx_repo_packages:
   - nginx
   - policycoreutils-python
 
+nginx_openssl_packages:
+  1.0.2: openssl
+  1.1.1: openssl11
```

Comment:
Do we really need to define the patch version here? Shouldn't be more reasonable to just define keys for `1.0` and `1.1`?

Reply:
Noup, you're right. I copied the name of the OpenSSL version 🙈